.github/workflows/release.yml

```diff
@@ -23,9 +23,15 @@ jobs:
         uses: cycjimmy/semantic-release-action@v4.1.0
         id: semantic
         env:
-          GITHUB_TOKEN: ${{ secrets.GH_RELEASE_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
 
       - name: Update README.md 📄
+        uses: actions/checkout@v4
         run: |
           sed -i "s/Documentation:.*/Documentation: [main](https:\/\/geostyler.github.io\/geostyler\/main\/index.html) \/ [latest](https:\/\/geostyler.github.io\/geostyler\/latest\/index.html) \/ [${{ steps.semantic.outputs.new_release_version }}](https:\/\/geostyler.github.io\/geostyler\/v${{ steps.semantic.outputs.new_release_version }}\/index.html)/g" README.md
+          git config user.name "github-actions[bot]"
+          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
Is this email static or can the number change?

Reply:
Taken from here: https://github.com/actions/checkout?tab=readme-ov-file#push-a-commit-using-the-built-in-token

Reply:
Not sure to be honest.